### ISSUE TITLE
Documentation: Running multiple projects

### ISSUE THREAD

**Issue by inspiredearth** [carlalexander/debops-wordpress#193]
Can is suggest adding some info to the documentation, pertaining to creating multiple projects.

In the mean time, am I correct to assume that to set up additional Debops-wordpress sites, I start from scratch each time? Creating a new root project folder, and following through the existing instructions? Or is there a way to have multiple sites in one project, and to deploy them independently?

**Comment by carlalexander** [comment]
Good call, I'll write documentation on that. I use one project for multiple sites.
